alert/types.go

```diff
@@ -125,6 +125,7 @@ const (
 	Warning
 	Critical
 	maxLevel
+	Error
```

Comment:
This is unused correct?

Reply:
I'll remove this one for now, but it is technically an option for the v2 API.

https://v2.developer.pagerduty.com/docs/send-an-event-events-api-v2

Since I'm not explicitly using it in the service then it's fine to ignore.

Reply:
Yeah, eventually Kapacitor may support custom alert levels, right now it tries to keep things simple by implementing what is common between most alert services. 

Once there is room for customization here, we can enable the pagerduty2 to take advantage of it.

Thanks.